update of STM32CubeF4 to version 1.9

rosterloh opened the pull request.

rosterloh committed:
update of STM32CubeF4 to version 1.8

rosterloh committed:
update of STM32CubeF4 to version 1.8

miklis commented:
Automatic CI verification build not done, please verify manually.

0xc0170 commented:
Thanks for updating the cube driver. 

We are close to the release, merging needs to wait at least until November. If there's any change, we will let you know. I'll pull this to test locally in the meantime.

rosterloh committed:
update of STM32CubeF4 to version 1.9

rosterloh committed:
update of STM32CubeF4 to version 1.9

rosterloh commented:
There has since been another cube update. Pretty trivial, only dsi changes. So I've updated this PR. Any chance of a merge?

betzw commented:
LGTM

0xc0170 commented:
We will test soon, to proceed with this Cube update

AlessandroA committed:
Updated dependency on latest uvisor-lib

uvisor-lib 2.0.0 was just released. This commit updates the dependency
on uvisor-lib to take into account this version.

0xc0170 committed:
Merge pull request #11 from AlessandroA/master

Updated dependency on latest uvisor-lib

0xc0170 committed:
version v1.1.0

dbestm committed:
[NUCLEO_F401RE] modify I2C driver for asynchronous

bcostm committed:
Add DMA fix file

Need to remove the half transfer interrupt in order to use the serial
communication with DMA.

bcostm committed:
Remove original DMA file

bcostm committed:
version v1.2.0

Commit:
Merge branch 'master' of https://github.com/rosterloh/mbed-hal-st-stm32cubef4

ciarmcom commented:
Automatic CI verification build not done, please verify manually.
